bitcoin/src/network.rs

```diff
@@ -52,27 +52,86 @@ impl From<Network> for NetworkKind {
 
         match n {
             Bitcoin => NetworkKind::Main,
-            Testnet | Signet | Regtest => NetworkKind::Test,
+            Testnet(_) | Signet | Regtest => NetworkKind::Test,
         }
     }
 }
 
-/// The cryptocurrency network to act on.
+/// The testnet version to act on.
 #[derive(Copy, PartialEq, Eq, PartialOrd, Ord, Clone, Hash, Debug)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "serde", serde(crate = "actual_serde"))]
 #[cfg_attr(feature = "serde", serde(rename_all = "lowercase"))]
 #[non_exhaustive]
+pub enum TestnetVersion {
+    /// Testnet version 3. This is the default testnet version.
```

Comment:
Why default? Wouldn't the latest one become the default eventually? Also it misleads users into thinking `Default` is implemented which it (rightly) isn't.

Reply:
Thank you very much for your suggestion. I will change this in time